### PR TITLE
Func DemoAlerts() - Refactoring

### PR DESCRIPTION
**Pull request by mlipok**
btw.
I also notice that `_WD_Alert($sSession, 'sendtext', 'new text')` doesn't work.
Could you confirm this issue ?

### CODE REVIEW

**Comment by mlipok** [comment]
All `_WD_***` functions should not be placed in `ConsoleWrite()`
Because `_WD_***` functions throws they own messages to Console so, `ConsoleWrite()` messages will be affected.

Thus this coding style should be used:
```AutoIt
$sText = _WD_Alert($sSession, 'gettext')
ConsoleWrite("- " & 'Text Detected => ' & $sText & @CRLF)
```

instead this kind of code:
```AutoIt
ConsoleWrite("- " & 'Text Detected => ' & _WD_Alert($sSession, 'gettext') & @CRLF)
```

**Comment by Danp2** [comment]
>  I also notice that _WD_Alert($sSession, 'sendtext', 'new text') doesn't work.

Looks like it is designed to work with `window.prompt` only, so the example needs to be updated.

**Comment by mlipok** [comment]
> Looks like it is designed to work with `window.prompt` only, so the example needs to be updated.


Please provide fix.

> You removed the initial status check that occurred before the Alert was shown. This was intended to be there to demonstrate how _WD_Alert responded when there wasn't a visible alert.

Please provide fix.

File: wd_demo.au3 [hunk 1]
```diff
@@ -292,6 +292,11 @@ EndFunc   ;==>DemoCookies
 Func DemoAlerts()
 	Local $sStatus, $sText
 
+	; check status before displaying Alert
+	; check status before displaying Alert
```
Comment (mlipok) [inline wd_demo.au3:296]: duplicated comments ?

Comment (Danp2) [reply wd_demo.au3]: Fixed

**Review by Danp2 — APPROVED** [review]

**Pull request merged**